### PR TITLE
Pin to v0.5.0 release.

### PR DESCRIPTION
**Pull request by Gerrit91**

### CODE REVIEW

File: inventories/group_vars/control-plane/metal.yml [hunk 1]
```diff
@@ -12,16 +12,16 @@ metal_api_admin_key: metal-admin
 metal_api_nsq_tcp_address: nsqd:4150
 
 metal_api_images:
-- id: firewall-ubuntu-2.0.20201126
-  name: Firewall 2 Ubuntu 20201126
-  description: Firewall 2 Ubuntu 20201126
-  url: https://images.metal-stack.io/metal-os/master/firewall/2.0-ubuntu/20201126/img.tar.lz4
+- id: firewall-ubuntu-2.0.20201214
```
Comment (majst01) [inline inventories/group_vars/control-plane/metal.yml:15]: This image is broken

File: inventories/group_vars/control-plane/metal.yml [hunk 2]
```diff
@@ -12,16 +12,16 @@ metal_api_admin_key: metal-admin
...
+  name: Firewall 2 Ubuntu 20201214
+  description: Firewall 2 Ubuntu 20201214
+  url: https://images.metal-stack.io/metal-os/master/firewall/2.0-ubuntu/20201214/img.tar.lz4
   features:
     - firewall
-- id: ubuntu-20.04.20201126
-  name: Ubuntu 20.04 20201126
-  description: Ubuntu 20.04 20201126
-  url: https://images.metal-stack.io/metal-os/master/ubuntu/20.04/20201126/img.tar.lz4
+- id: ubuntu-20.04.20201214
```
Comment (majst01) [inline inventories/group_vars/control-plane/metal.yml:21]: also

File: inventories/group_vars/control-plane/metal.yml (hunk 1 above)
Comment (Gerrit91) [reply inventories/group_vars/control-plane/metal.yml:15]: Mh... it's not so good that we are releasing broken images. :( 

Proper releases with notes and things would be nice to for the images.

Will revert this.

Comment (majst01) [reply inventories/group_vars/control-plane/metal.yml]: Sadly we have no proper release process for images.

**Comment by Gerrit91** [comment]
Gonna merge this now with Admin privileges, review is not so important on this one, I guess.

**Pull request merged**